pai-management/k8sPaiLibrary/maintainlib/add.py

```diff
@@ -55,32 +55,24 @@ def __init__(self, cluster_config, node_config, clean):
         else:
             self.jobname = "error"
             self.logger.error("[{0}] Error: {1} is an undefined role, quit add job in host [{2}]".format(time.asctime(), node_config['k8s-role'], node_config['nodename']))
+            sys.exit(1)
 
 
 
     def prepare_package(self, node_config, job_name):
 
-        if self.jobname == 'error':
-            return
-
```

Comment:
Hi @ydye ,

When would `self.jobname == 'error'`?

Reply:
In the function init(), this error has been detected and exit from the program.